homeassistant/components/group.py

```diff
@@ -112,6 +113,11 @@ def setup(hass, config):
 
 class Group(object):
```

Comment:
Wouldn't adding it to [the Entity class](https://github.com/balloob/home-assistant/blob/master/homeassistant/helpers/entity.py#L15) and have Group extend from Entity it should just work ? 

All entities should already be extend from the Entity class.